VitalClient/src/main/java/io/tryvital/client/utils/VitalRequestInterceptor.kt

```diff
@@ -0,0 +1,58 @@
+package io.tryvital.client.utils
+
+import okhttp3.*
+import okio.Buffer
+import okio.BufferedSink
+import okio.GzipSink
+import okio.buffer
+
+const val VITAL_SDK_VERSION = "0.9.4"
+
+internal class VitalRequestInterceptor : Interceptor {
+    override fun intercept(chain: Interceptor.Chain): Response {
+        val request = chain.request()
+        var builder = request.newBuilder()
+            .header("x-vital-ios-sdk-version", VITAL_SDK_VERSION)
```

Comment:
We shouldn't be using the `ios` infix here.

Reply:
> subject to change; added for development testing